includes/class-server-rules.php

```
@@ -0,0 +1,117 @@
+				'    text/css',
+				'    text/plain',
+				'    text/x-component;',
+				'',
+				'# Browser Caching',
+				'location ~* \.(jpg|jpeg|gif|png|webp|avif|svg|woff|woff2|ttf|otf|eot|ico|css|js)$ {',
+				'    expires 365d;',
+				'    add_header Cache-Control "public, no-transform";',
+				'    access_log off;',
+				'}',
+			) );
+		}
```

PHPMD (2.15.0)

[error] The method get_nginx_rules is not named in camelCase. (undefined)

(CamelCaseMethodName)